src/czar/Czar.cc

```diff
@@ -99,7 +99,7 @@ void Czar::_monitor() {
 
         /// Check database for changes in worker chunk assignments and aliveness
         try {
-            // TODO:UJ The read() is incredibly expensive until the database has
+            // TODO:DM-53239 The read() is incredibly expensive until the database has
```

Comment:
Noted! We should find a solution to this problem. Please, ping me when you feel this would be the blocking factor.